packages/serverless/src/domain/use-case/document-history.ts

```diff
@@ -64,6 +69,23 @@ function buildHistoryPipeline(pagination: TPagination): Array<any> {
   ];
 }
 
+// TODO: need debugging to confirm the accuracy of this type annotation
+type TDocumentHistorySerialized = {
```

Comment:
Type is wrong, please check `TMetadataDetail`

Reply:
fixed